libs/shared/ui/src/lib/components/inputs/input-select/input-select.tsx

```diff
@@ -1,125 +1,170 @@
-import { ListboxButton, ListboxInput, ListboxList, ListboxOption, ListboxPopover } from '@reach/listbox'
 import { useEffect, useState } from 'react'
+import Select, {
+  GroupBase,
+  MultiValue,
+  MultiValueProps,
+  OptionProps,
+  SingleValue,
+  SingleValueProps,
+  components,
+} from 'react-select'
 import { Value } from '@console/shared/interfaces'
 import Icon from '../../icon/icon'
-import InputSearch from '../input-search/input-search'
 
 export interface InputSelectProps {
-  label: string
-  value?: string | undefined
-  items: Value[]
   className?: string
-  onChange?: (e: string) => void
-  error?: string
+  label: string
+  value?: string | string[]
+  options: Value[]
   disabled?: boolean
-  portal?: boolean
+  error?: string
+  onChange?: (e: string | string[]) => void
   dataTestId?: string
-  search?: boolean
+  isMulti?: true
+  portal?: boolean
+  isSearchable?: boolean
+  isClearable?: boolean
 }
 
 export function InputSelect(props: InputSelectProps) {
   const {
+    className = '',
     label,
     value,
-    items,
-    className = '',
+    options,
+    disabled,
+    error = false,
     onChange,
-    error,
     dataTestId,
-    disabled = false,
-    portal = true,
-    search,
+    isMulti = undefined,
+    isSearchable = false,
+    isClearable = false,
   } = props
+  const [focused, setFocused] = useState(false)
+  const [selectedItems, setSelectedItems] = useState<MultiValue<Value> | SingleValue<Value>>([])
+  const [selectedValue, setSelectedValue] = useState<string | string[]>([])
 
-  const [selectedValue, setSelectedValue] = useState(value && items.find((item) => item.value === value))
-  const hasError = error && error.length > 0 ? 'input--error' : ''
+  const hasFocus = focused
+  const hasError = error ? 'input--select-multiple--error' : ''
```

Comment:
I see on this component a mix between `input--select` and `input--select-multiple`, it's possible to remove the `input-select-multiple` or add it when you need a specific class for multiple?